mlrun/datastore/targets.py

```diff
@@ -1237,18 +1289,20 @@ def _get_target_path(driver, resource):
         else "vectors"
     )
     name = resource.metadata.name
-    version = resource.metadata.tag
     project = resource.metadata.project or mlrun.mlconf.default_project
     data_prefix = get_default_prefix_for_target(kind).format(
-        project=project, kind=kind, name=name
+        project=project,
+        kind=kind,
+        name=name,
+        run_uuid=mlrun.mlconf.feature_store.run_uuid_place_holder,
```

Comment:
Not sure I understand this - here you're basically replacing `{run_uuid}` with the string defined as the place-holder which is `{run_uuid}`. So the string you get at the end will still contain `{run_uuid}` in it. I understand this is because `PathObject` will later do the run_uuid translation, but note that `PathObject` does not respect the config place-holder - it looks explicitly for `run_uuid`, so if the place-holder is different, this will not work eventually. I think the issue is probably not here, but rather in the `PathObject`, but I'm keeping this comment just to verify I understand what's happening here.

Reply:
Correct, when using format we must use all the place holders otherwise it breaks, that's why it is replacing it with itself.
As for the place holder definition it's not for it to be easily changed but just that it will be held in one place - we shouldn't change it, it will break many things and also will not be backward compatible. Added it as a comment above the definition.

Reply:
Putting a value in the config that can never be changed is bad practice. If you're using the global config as just a place to put constants, then I think you should place this in another location that makes sense and is accessible through all the clients that require access to this constant. The config is not meant for this purpose and it's error prone since it allows for external overrides which will be disastrous in this case.